tests/pyfunc/pyfunc_sample_code_with_config.py

```diff
@@ -0,0 +1,13 @@
+from mlflow.models import ModelConfig, set_model
+from mlflow.pyfunc import PythonModel
+
+base_config = ModelConfig(development_config="tests/pyfunc/pyfunc_sample_config.yml")
+
+
+class MyModel(PythonModel):
+    def predict(self, context, model_input):
+        timeout = base_config.get("timeout")
+        return f"Predict called with input {model_input}, timeout {timeout}"
+
+
+set_model(MyModel())
```

Comment:
What would happen if this line was missing?

Reply:
Yes, it errors out with 
```
MlflowException: `python_model` must be a PythonModel instance, filepath, or a callable object
```
I can update this with a more informative error message.